src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -126,15 +126,15 @@ RUN for iter in {1..10}; do \
       apt-get update  && \
       apt-get upgrade -y  && \
       apt-get install -y --no-install-recommends \
-       fontconfig libnss3 curl ca-certificates && \
+      fontconfig fonts-liberation libnss3 curl ca-certificates && \
       apt-get clean && \
       rm -rf /var/lib/apt/lists/* && exit_code=0 && break || exit_code=$? && echo "apt-get error: retry $iter in 10s" && \
       sleep 10; \
     done; \
     (exit $exit_code)
 {{/ubuntu}}
 {{#wolfi}}
-RUN apk --no-cache add bash curl fontconfig libstdc++ libnss findutils shadow ca-certificates
+RUN apk --no-cache add bash curl fontconfig fonts-liberation libstdc++ libnss findutils shadow ca-certificates
```

Comment:
`wolfi` is only needed for cloud and serverless. I know reporting is disabled for serverless, not sure about cloud.

Reply:
it's true reporting is disabled on serverless but it's enabled on cloud.